content/handbook/marketing/brand-and-product-marketing/content/editorial-team.md

```
@@ -19,295 +19,298 @@ stale: false
+「and」の代用として使用しないでください。
 
 ## ブランチ名
 
-ブランチの名前を書くとき、ベストプラクティスは引用符や斜体などではなく、バッククォートでスタイル設定することです。例: `sk-branch-name`
+ブランチの名前を書くとき、引用符やイタリック体などではなくバッククォートでスタイルするのがベストプラクティスです。例: `sk-branch-name`
 
-## 大文字化
+## 大文字の使用
 
-特定の単語のスタイル設定については[以下](#word-list)を参照してください。
+特定の単語のスタイリングについては、[以下](#word-list) を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)